source/documentation/API-reference.html.md.erb

```diff
@@ -9,7 +9,7 @@ weight: 50
 # API reference
 
 <div class="govuk-inset-text">
-This API is in Alpha and may have breaking changes.
+This API is in Beta and may have breaking changes.
```

Comment:
do we specify that its public or private Beta?

Reply:
Simon said it should say Beta